packages/back-end/src/routers/organizations/organizations.controller.ts

```diff
@@ -689,11 +689,23 @@ export async function getOrganization(
   if (licenseKey || process.env.LICENSE_KEY) {
     // automatically set the license data based on org license key
     license = getLicense(licenseKey || process.env.LICENSE_KEY);
-    if (!license || (license.organizationId && license.organizationId !== id)) {
+    if (
+      !license ||
+      (license.organizationId && license.organizationId !== id) ||
+      (license && !license.stripeSubscription) ||
+      (license && !license.orbSubscription)
+    ) {
       try {
         license =
-          (await licenseInit(org, getUserCodesForOrg, getLicenseMetaData)) ||
-          null;
+          (await licenseInit(
+            org,
+            getUserCodesForOrg,
+            getLicenseMetaData,
+            // if license is set, but no stripe or orb subscription, force refresh
+            !license?.stripeSubscription || !license?.orbSubscription
+              ? true
+              : false
+          )) || null;
```

Comment:
This is all getting a bit hard to follow. In `CloudProUpgradeModal` when a user successfully subscribes, this code, paired with the `refreshOrganization` call means the org's subscription status updates automatically, which is handy if they're upgrading from an upgrade CTA to get a certain feature (ex: Bandits).